### PR TITLE
New noWMS:world_depth_view for depth service replacement

### PR DESCRIPTION
**Pull request by pmbohm**
Good to go now using parameterisedSQL view
Responds to requests such as: http://geoserver-static.aodn.org.au/geoserver/wfs?typeName=noWMS:world_depth_view&SERVICE=WFS&REQUEST=GetFeature&VERSION=1.0.0&viewparams=lon:105.4248046875;lat:-19.8642578125

### CODE REVIEW

**Comment by pmbohm** [comment]
For https://github.com/aodn/backlog/issues/748

File: workspaces/noWMS/namespace.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<namespace>
+  <id>NamespaceInfoImpl-459f1a44:1624bc3a9cf:-7ffb</id>
+  <prefix>noWMS</prefix>
```
Comment (jonescc) [inline workspaces/noWMS/namespace.xml]: Is this still a good name?

Comment (pmbohm) [reply workspaces/noWMS/namespace.xml]: I'm thinking the work to hide layers will still be done.

**Comment by pmbohm** [comment]
Requires https://github.com/aodn/chef-private/pull/2753

**Comment by jonescc** [comment]
Looks good!

**Pull request merged**